clients/ui/frontend/src/shared/components/design/utils.ts

```diff
@@ -16,38 +34,276 @@ export enum SectionType {
 }
 
 export enum ProjectObjectType {
+  project = 'project',
```

Comment:
Do we need all of these

Reply:
Yes, leaving this in shared library since we'll soon need all these sections.